[rospy.client][INFO] 2021-01-16 08:40:09,063: init_node, name[/train_1502_1610786409034], pid[1502]
[xmlrpc][INFO] 2021-01-16 08:40:09,063: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-01-16 08:40:09,064: Started XML-RPC server [http://192.168.0.191:41501/]
[rospy.init][INFO] 2021-01-16 08:40:09,064: ROS Slave URI: [http://192.168.0.191:41501/]
[rospy.impl.masterslave][INFO] 2021-01-16 08:40:09,065: _ready: http://192.168.0.191:41501/
[rospy.registration][INFO] 2021-01-16 08:40:09,066: Registering with master node http://192.168.0.126:11311
[xmlrpc][INFO] 2021-01-16 08:40:09,066: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-01-16 08:40:09,165: registered with master
[rospy.rosout][INFO] 2021-01-16 08:40:09,165: initializing /rosout core topic
[rospy.rosout][INFO] 2021-01-16 08:40:09,171: connected to core topic /rosout
[rospy.simtime][INFO] 2021-01-16 08:40:09,173: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2021-01-16 08:40:09,189: topic[/bev/temporal_bev_image] adding connection to [http://192.168.0.126:36903/], count 0
[rospy.internal][INFO] 2021-01-16 08:40:09,204: topic[/start_goal_points] adding connection to [http://192.168.0.126:40949/], count 0
[rospy.internal][INFO] 2021-01-16 08:40:09,207: topic[/scan] adding connection to [http://192.168.0.126:45431/], count 0
[rospy.internal][INFO] 2021-01-16 08:40:09,211: topic[/is_start_episode] adding connection to [http://192.168.0.126:34007/], count 0
[rospy.internal][INFO] 2021-01-16 08:40:09,212: topic[/scan] adding connection to [http://192.168.0.126:36669/], count 1
[rospy.internal][INFO] 2021-01-16 08:40:09,213: topic[/odom] adding connection to [http://192.168.0.126:45431/], count 0
[rospy.internal][INFO] 2021-01-16 08:40:09,366: topic[/cmd_vel] adding connection to [/gazebo], count 0
[rospy.internal][INFO] 2021-01-16 08:40:09,367: topic[/episode_watcher/is_finish_episode] adding connection to [/episode_manager], count 0
[rospy.internal][INFO] 2021-01-16 08:40:09,465: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2021-01-16 08:40:11,701: signal_shutdown [atexit]
[rospy.internal][INFO] 2021-01-16 08:40:11,704: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-01-16 08:40:11,705: topic[/cmd_vel] removing connection to /gazebo
[rospy.internal][INFO] 2021-01-16 08:40:11,705: topic[/episode_watcher/is_finish_episode] removing connection to /episode_manager
[rospy.internal][INFO] 2021-01-16 08:40:11,706: topic[/bev/temporal_bev_image] removing connection to http://192.168.0.126:36903/
[rospy.internal][INFO] 2021-01-16 08:40:11,706: topic[/odom] removing connection to http://192.168.0.126:45431/
[rospy.internal][INFO] 2021-01-16 08:40:11,707: topic[/start_goal_points] removing connection to http://192.168.0.126:40949/
[rospy.internal][INFO] 2021-01-16 08:40:11,707: topic[/scan] removing connection to http://192.168.0.126:45431/
[rospy.internal][INFO] 2021-01-16 08:40:11,708: topic[/scan] removing connection to http://192.168.0.126:36669/
[rospy.internal][INFO] 2021-01-16 08:40:11,709: topic[/is_start_episode] removing connection to http://192.168.0.126:34007/
[rospy.internal][WARNING] 2021-01-16 08:40:11,709: Unknown error initiating TCP/IP socket to 192.168.0.126:43067 (http://192.168.0.126:40949/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.impl.masterslave][INFO] 2021-01-16 08:40:11,709: atexit
[rospy.internal][WARNING] 2021-01-16 08:40:11,710: Unknown error initiating TCP/IP socket to 192.168.0.126:59449 (http://192.168.0.126:36903/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

